pages/linux/objcopy.md

```diff
@@ -0,0 +1,24 @@
+# objcopy
+
+> Copy the contents of an object file to another.
+> More information: <https://manned.org/objcopy>.
+
+- Copy all data from a specific file to another one:
+
+`objcopy {{path/to/file/source_file}} {{path/to/file/target_file}}`
+
+- Translate object files from one format to another:
+
+`objcopy --input-target={{inputformat}} --output-target={{output_format}} {{path/to/your/source_file}} {{path/to/your/target_file}}`
+
+- Strip all symbol information from a specific file:
+
+`objcopy --strip-all {{path/to/your/source_file}} {{path/to/your/target_file}}`
+
+- Strip debugging information from file:
+
+`objcopy --strip-debug {{/path/to/your/source_file}} {{path/to/your/target_file}}`
```

Comment:
```suggestion
`objcopy --strip-debug {{path/to/source_file}} {{path/to/target_file}}`
```
Remember to not start file placeholders with "/" if both absolute and relative paths are allowed.

As for "your", see my previous comment.